bigtable-hbase-2.x-parent/bigtable-hbase-2.x/src/main/java/com/google/cloud/bigtable/hbase2_x/BigtableAsyncBufferedMutator.java

```diff
@@ -102,15 +105,7 @@ public long getWriteBufferSize() {
    */
   @Override
   public CompletableFuture<Void> mutate(final Mutation mutation) {
-    return FutureUtils.toCompletableFuture(helper.mutate(mutation)).thenApply(r -> null);
-  }
-
-  /**
-   * <p>hasInflightRequests.</p>
-   *
-   * @return a boolean.
-   */
-  public boolean hasInflightRequests() {
-    return helper.hasInflightRequests();
+    return toCompletableFuture(helper.mutate(mutation))
+        .thenApply(r -> null);
   }
-}
+}
```

Comment:
Can we please have a `newline` character here.

Reply:
done